### ISSUE TITLE
On the homepage make the 4 cards as the link of the respective pages

### ISSUE THREAD

**Issue by deepanshu-sharmaa** [deepanshu-sharmaa/Red-Saviour#1]

**Comment by deepanshu-sharmaa** [comment]
Go ahead @techsnap. Happy Contributing.

**Comment by viveena30** [comment]
could you assign me this issue? @deepanshu-sharmaa

**Comment by techsnap** [comment]
Hey @deepanshu-sharmaa , should I keep old `<a>` tag inside the card as it is or I can remove `<a>` tag to make the whole card link to another webpage? Refer this screenshot:

![index html](https://user-images.githubusercontent.com/62293137/194153533-8514fab4-f5fa-45fb-bba5-ca859ab4ff0c.png)

I have modified 1st card to make the card link to another page but the card heading now does not contain `<a>` tag. Instead, I have wrapped the whole `<div>` inside `<a>`. Now, card heading also appears as white instead of blue. Is it okay or should I keep the color same?
